### PR TITLE
[17.0] helpdesk_mgmt

### PR DESCRIPTION
**Pull request by chandni-serpentcs**

### CODE REVIEW

**Comment by dalonsod** [comment]
Hello @chandni-serpentcs could you take in account #474 recently merged? Thanks!

**Comment by carolinafernandez-tecnativa** [comment]
Hi we just added this changes, could you please check it and added to this migration? https://github.com/OCA/helpdesk/pull/534

**Comment by carolinafernandez-tecnativa** [comment]
Hi ! could you please take into account this fix? in order to avoid getting error message described on issue.

https://github.com/OCA/helpdesk/pull/541#pullrequestreview-1850434719

**Comment by victoralmau** [comment]
Please, include https://github.com/OCA/helpdesk/pull/542

**Comment by victoralmau** [comment]
Please, cherry-pick https://github.com/OCA/helpdesk/pull/563 to commit history.

**Comment by pedrobaeza** [comment]
There are several improvements made on previous version that needs to be cherry-picked here. Are you going to continue this PR?

**Comment by marcelsavegnago** [comment]
@chandni-serpentcs can you rebase please ?

**Comment by victoralmau** [comment]
Please, cherry-pick https://github.com/OCA/helpdesk/pull/557 + https://github.com/OCA/helpdesk/pull/604 to commit history.

**Comment by victoralmau** [comment]
Please cherry-pick https://github.com/OCA/helpdesk/pull/621 to commit history

**Comment by yvaucher** [comment]
@chandni-serpentcs Can you work on this? At least one other PR depends on this one.

File: helpdesk_mgmt/data/helpdesk_data.xml [hunk 1]
```diff
@@ -320,7 +320,7 @@
         <record id="mail_alias_lead_info_helpdesk" model="mail.alias">
             <field name="alias_name">help</field>
             <field name="alias_model_id" ref="model_helpdesk_ticket" />
-            <field name="alias_user_id" ref="base.user_admin" />
+<!--            <field name="alias_user_id" ref="base.user_admin" />-->
```
Comment (giarve) [inline helpdesk_mgmt/data/helpdesk_data.xml:323]: Reason for leaving this commented? why not remove entirely if not working or fixing it with the correct record?

EDIT: field does not exist anymore in 17.0

File: helpdesk_mgmt/views/helpdesk_ticket_team_views.xml [hunk 2]
```diff
@@ -87,7 +87,7 @@
                                 class="oe_inline"
                                 string="Accept Emails From"
                             />
-                            <field name="alias_user_id" string="Assign to" />
+<!--                            <field name="alias_user_id" string="Assign to" />-->
```
Comment (giarve) [inline helpdesk_mgmt/views/helpdesk_ticket_team_views.xml:90]: also commented out

**Comment by giarve** [comment]
I want to make it explicit that previous PR commits were not cherry picked but introduced as authored commits. If anybody wants to open up a new PR for this. Some of the commits made by the author are still valid, specially the pre-commit fixes one (could be cherry-picked).

**Comment by pedrobaeza** [comment]
Superseded by #638

**Pull request closed without merging**